libexec/bats-core/bats-exec-test

```diff
@@ -365,6 +370,8 @@ bats_evaluate_preprocessed_source() {
   if [[ -z "$BATS_TEST_SOURCE" ]]; then
     BATS_TEST_SOURCE="${BATS_PARENT_TMPNAME}.src"
   fi
+  # Dynamically loaded user files provided outside of Bats.
+  # shellcheck disable=SC1090
```

Comment:
https://github.com/koalaman/shellcheck/wiki/SC1090
Ignoring as we cannot include files that do not exist yet.